infra/prod/_modules/functions/private_endpoint_relying_party.tf

```diff
@@ -0,0 +1,20 @@
+resource "azurerm_private_endpoint" "relying_party" {
```

Comment:
I'd add a comment here to explain where we use this one.

Reply:
[be2f437](https://github.com/pagopa/io-fims/pull/18/commits/be2f4374778580c8e9390e4103952e1bfd5c4971)